Dockerfile.debian

```diff
@@ -0,0 +1,102 @@
+# syntax = docker/dockerfile:1.2
+FROM docker.io/library/golang:1.19-bullseye AS builder
+
+RUN apt update
+RUN apt install -y build-essential git bash ca-certificates libstdc++6
+
+WORKDIR /app
+ADD go.mod go.mod
+ADD go.sum go.sum
+
+RUN go mod download
+ADD . .
+
+RUN --mount=type=cache,target=/root/.cache \
+    --mount=type=cache,target=/tmp/go-build \
+    --mount=type=cache,target=/go/pkg/mod \
+    make all
+
+
+FROM docker.io/library/golang:1.19-alpine3.16 AS tools-builder
+
+RUN apk --no-cache add build-base linux-headers git bash ca-certificates libstdc++
+WORKDIR /app
+
+ADD Makefile Makefile
+ADD tools.go tools.go
```

Comment:
I think this line is useless, because tools.go used to ensure binary dep is stored in go.mod

Reply:
since i'm not copying the rest of the files, 'go mod vendor' will run 'go mod tidy' first, which cause those deps to be removed, unless tools.go file is there

Reply:
Don't see reason to call "go mod tidy" in docker.

Reply:
'go mod vendor' which is used by the db-tools build process calls 'tidy'. not sure if there's a way to call vendor without tidy

Reply:
got it